site/de/_posts/2021-06-10-projects.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+lang: de
+lang-ref: Bewilligung OCR-D-Projekte
+title: "Implementierungs- und Modulprojekte bewilligt"
+tags: [DFG, grant, phase 3]
+author: Elisabeth Engl
+---
+
+Neben dem Antrag des Koordinierungsprojekts wurden von der DFG sieben Implementierungs- 
+und Modulprojekte bewilligt, die in den kommenden Monaten ihre Arbeit aufnehmen werden.
+
+
+Von den elf eingereichten Anträgen, die im Zuge einer umfangreichen Pilotierung der 
+OCR-D-Software im Sommer 2020 erarbeitet wurden, werden vier Implementierungs- und drei 
+Modulprojekte von der DFG gefördert. 
```

Comment:
Gehören die Bedarfsfallprojekte zu den Modulprojekten oder sollten sie ggf. separat erwähnt werden?

Reply:
that's the very same, no need to mention it separately